src/components/payment/payment.html

```diff
@@ -0,0 +1,55 @@
+<transition name="modal">
+<div class="modal-mask" @click="close">
+    <div class="modal-container" @click.stop>
+      <span @click="close" class="close">&times;</span>
+      <div class="modal-header">
+	<center>
+          <h3> Buy more space</h3>
+	</center>
+      </div>
+
+      <div class="modal-body">
+	<center>
+          <div v-bind:class="errorClass">
+            <label v-if="isError">{{ error }}</label>
+	    <br/>
+	    <label> Current space: {{ getQuota() }}</label>
+	    <br/>
+            <button class="btn btn-success" @click="requestStorage(10737418240)" style="height:10em;">
+              <div style="font-size:x-large">Basic account</div>
+	      <br/>
+	      <div>
+		<ul style="text-align:left;padding-right:40px;">
+		  <li>10 GB of hyper secure storage</li>
+		  <li>5 GBP / month</li>
+		</ul>
+	      </div>
+            </button>
+	    <button class="btn btn-success" @click="requestStorage(107374182400)" style="height:10em;">
+              <div style="font-size:x-large">Visionary account</div>
```

Comment:
3 options maybe?

Reply:
Yep, the standard advice is to do 3 options, but what would our third be? We can't support 1 TiB level yet.